publications.md

```diff
@@ -0,0 +1,459 @@
+*   Lago, Veronique, Susan E. Wijffels, **Paul J. Durack**, John A. Church, Nathaniel, L. Bindoff and Simon J. Marsland, 2016: Simulating the role of surface forcing on observed multidecadal upper ocean salinity changes, _Journal of Climate_, **29** (15), 5575-5588 doi: [10.1175/JCLI-D-15-0519.1]({{site.baseurl}}/staff/durack/links/Lagoetal16JCL.html)
+*   Annamalai, H. and **K. R. Sperber**, 2016: South Asian summer monsoon variability in a changing climate. Chapter 3 (pp. 25-46) in The Monsoons and Climate Change. Eds. L. M. V. Carvalho and C. Jones, Springer International Publishing, Switzerland, 253pp.
+*   **Santer, B.D**., S. Solomon, G. Pallotta, C. Mears, S. Po-Chedley, Q. Fu, F.J. Wentz, C.-Z. Zou, **J.F. Painter**, I. Cvijanovic, and **C. Bonfils**, 2016: Comparing tropospheric warming in climate models and satellite data. _Nat. Clim. Change_ (submitted).
+*   Griffies, S. M., G. Danabasoglu, **P. J. Durack**, A. J. Adcroft, V. Balaji, C. Boening, E. P. Chassignet, E. Curchitser, J. Deshayes, H. Drange, B. Fox-Kemper, **P. J. Gleckler**, J. M. Gregory, H. Haak, R. W. Hallberg, H. T. Hewitt, D. M. Holland, T. Ilyina, J. H. Junglaus, Y. Komuro, J. P. Krasting, W. G. Large, S. J. Marsland, S. Masina, T. J. McDougall, J. C. Orr, A. Pirani, F. Qiao, R. J. Stouffer, **K. E. Taylor**, H. Tsujino, P. Uotila, M. Valdivieso, M. Winton and S. G. Yeager, (2016): Experimental and diagnostic protocol for the physical component of the CMIP6 Ocean Model Intercomparison Project (OMIP). _Geoscientific Model Development_, **9**, 3231-3296. doi: [10.5194/gmd-9-3231-2016](http://doi.org/10.5194/gmd-9-3231-2016)
+*   Meehl, G.A., A. Hu, **B.D. Santer**, and S.-P. Xie, 2016: Interdecadal Pacific Oscillation contributions to multi-decadal variability of 20th century globally averaged surface temperatures
+*   Fyfe, J.C., G.A. Meehl, M.H. England, M.E. Mann, **B.D. Santer**, G.M. Flato, E. Hawkins, N.P. Gillett, S.-P. Xie, Y. Kosaka, and N.C. Swart, 2016: Making sense of the early 2000s global warming slowdown, _Nature Climate Change_, **6**, 224-228. doi:[10.1038/nclimate2938](http://dx.doi.org/10.1038/nclimate2938)
+*   **Ivanova, D. P., P. J. Gleckler, K. E. Taylor, P. J. Durack, K. D. Marvel**, 2016: Moving beyond the total sea ice extent in gauging model biases, _Journal of Climate_, **29**, 8965-8987. doi:[10.1175/JCLI-D-16-0026.1](http://dx.doi.org/10.1175/JCLI-D-16-0026.1)
+*   Clark, P.U., J.D. Shakun, S.A. Marcott, A.C. Mix, M. Eby, S. Kulp, A. Levermann, G.A. Milne, P.L. Pfister, **B.D. Santer**, D.P. Schrag, S. Solomon, T.F. Stocker, B.H. Strauss, A.J. Weaver, R. Winkelmann, D. Archer, E. Bard, A. Goldner, K. Lambeck, R.T. Pierrehumbert, and G.-K. Plattner, 2016: Consequences of 21st century policy for multi-millennial climate and sea-level change, _Nature Climate Change_, **6**, 360-369
```

Comment:
```suggestion
*   Clark, P.U., J.D. Shakun, S.A. Marcott, A.C. Mix, M. Eby, S. Kulp, A. Levermann, G.A. Milne, P.L. Pfister, **B.D. Santer**, D.P. Schrag, S. Solomon, T.F. Stocker, B.H. Strauss, A.J. Weaver, R. Winkelmann, D. Archer, E. Bard, A. Goldner, K. Lambeck, R.T. Pierrehumbert, and G.-K. Plattner, 2016: Consequences of 21<sup>st</sup> century policy for multi-millennial climate and sea-level change, _Nature Climate Change_, **6**, 360-369
```

Reply:
@mauzey1 

Good to catch these.   Do I need to do anything to make sure they get incorporated?

Reply:
@mauzey1 

Thanks for the tip about "Commit".  I will do so, if the situation arises again.